ci/docker/runtime_functions.sh

```diff
@@ -163,6 +163,30 @@ build_android_arm64() {
     cp dist/*.whl /work/build
 }
 
+build_android_arm64() {
+    set -ex
+    cd /work/build
+#        -DCMAKE_ANDROID_NDK=${CROSS_ROOT} \
+#        -DCMAKE_SYSTEM_VERSION=${ANDROID_NDK_REVISION} \
+#        -DCMAKE_SYSTEM_NAME=Android \
+    cmake\
+        -DANDROID=ON \
+        -DUSE_CUDA=OFF\
+        -DUSE_SSE=OFF\
+        -DUSE_LAPACK=OFF\
+        -DUSE_OPENCV=OFF\
+        -DUSE_OPENMP=OFF\
+        -DUSE_SIGNAL_HANDLER=ON\
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo\
```

Comment:
To reduce the size, I'd propose we build without debug info for now

Reply:
I want symbols to diagnose initial runs and crashes, there's a reason for this.

Reply:
I'd propose that you set that flag locally then. We're referencing these instructions in our guides and to our customers, means we have to provide a ready-made solution. We had the same case previously when a customer complained about the size and in fact it was only the RelWithDebInfo.

Reply:
I don't see how the points you are making about an experimental build which is not official nor tested are of any relevance at this stage of Android support. This is a development build and I want debug symbols to diagnose crashes, this should be reason enough to have it as it is, because size of build doesn't really matter right now, and debugability is more important.

Reply:
We already spoke about this offline and I said that I'm fine with moving forward.